platform/linux-generic/include/odp/api/plat/event_validation_external.h

```diff
@@ -42,12 +42,19 @@ typedef enum {
 	_ODP_EV_PACKET_FREE_MULTI,
```

Comment:
In 3e25029cc1bb1730cfa1fc43ca3b00f6a69c8e56 message, there could be a similar list of affected functions as in previous commit.

Reply:
Fixed in V4.